package.json

```diff
@@ -102,7 +102,6 @@
     "Braze"
   ],
   "dependencies": {
-    "debug": "^4.3.2",
-    "lodash": "^4.17.21"
+    "debug": "^4.3.2"
```

Comment:
Nit: maybe we can in another PR remove depenedency on debug? https://bundlephobia.com/package/debug@4.3.4